scripts/complete_index.py

```diff
@@ -0,0 +1,73 @@
+"""
+Script to identify and complete general index entries with original content.
+
+This script processes .po files, identifies out-of-order entries based on the
+source order, and completes them with original content.
+
+Usage:
+    python script_name.py [list of .po files]
+
+If no list of .po files is provided, the script processes all .po files in the
+current directory and its immediate subdirectories.
+"""
+
+from pathlib import Path
+import sys
+
+import polib
+
+
+def out_of_order_entries(po_file):
+    """
+    Compare the order of source lines against the order in which they appear in
+    the file, and return a generator with entries that are out of order.
+    """
+    po_file = polib.pofile(po_file)
+    po_entries = [entry for entry in po_file if not entry.obsolete]
+    val_max = 0
+
+    for entry in po_entries:
+        source_index = int(entry.occurrences[0][1])
+
+        if source_index <= val_max:
+            yield(entry)
```

Comment:
`yield` no es una función, sino que una expresión, así que lo que estás haciendo en verdad acá es `yield` `(entry)` (hacer yield de entry entre paréntesis). Funciona, pero es confuso.

```suggestion
            yield entry
```